sonar-batch-protocol/src/main/protobuf/batch_report.proto

```diff
@@ -83,9 +83,7 @@ message Component {
   optional string description = 16;
 
   // temporary fields during development of computation stack
-  optional int64 id = 13;
```

Comment:
as long as this protobuf file does not support past versions, field ids can be compressed (no holes in sequence of numbers)